jenkins/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM mirrors--dockerhub.eu-west-1.artifactory.aws.arm.com/ubuntu:20.04
```

Comment:
Pass this in as a variable rather than exposing internal paths?

Reply:
More generally, do we want to put these internal build pipelines into public repo?

Reply:
Or can we build them with Github actions?